client/ayon_houdini/api/plugin.py

```diff
@@ -334,6 +348,34 @@ class HoudiniInstancePlugin(pyblish.api.InstancePlugin):
     hosts = ["houdini"]
     settings_category = SETTINGS_CATEGORY
 
+    def evalParmNoFrame(self, rop, parm, **kwargs):
+        try:
+            return evalParmNoFrame(rop, parm, **kwargs)
+        except Exception as exc:
+            raise PublishError(
+                f"Failed evaluating parameter '{parm}' on Rop node: {rop.path()}",
+                detail=f"{exc}"
```

Comment:
The `f{exc}` seems weird.
I assume this doesn't work?
```suggestion
                detail=exc
```

If so, why not do:
```suggestion
                detail=str(exc)
```

Reply:
https://github.com/ynput/ayon-houdini/pull/190/commits/a03e49bc665970ec532140908df82604537e7ae6